spec/date.html

```diff
@@ -468,7 +470,7 @@ <h1>Properties of Temporal.Date Instances</h1>
   </emu-clause>
 
   <emu-clause id="sec-temporal-date-abstract-ops">
-    <h1>Abstract operations</h1>
+    <h1>Abstract Operations for Temporal.Date Objects</h1>
```

Comment:
None of the other abstract ops sections make this explicit; do we want to be consistent?

Reply:
That's because I didn't push changes to those! I'll update these everywhere to ultimately make the spec consistent with ECMA 262.